index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    <script defer src="./scripts.js"></script>
+
+    <title>Avengers Conference</title>
+</head>
+
+<body>
+    <header>
+
+        <div class="black-bar">
+            <div class="container top-dark">
+                <ul class="top-dark-icons">
+                    <li><img src="./images/Logos/face.png" alt="logo" /></li>
+                    <li><img src="./images/Logos/twiiter.png" alt="logo" /></li>
+                    <li><a href="#program " onClick="menuClose(1);">English</a></li>
+                    <li><a href="#speakers " onClick="menuClose(1);">MyPage</a></li>
+                    <li><a href="#sponsors" onClick="menuClose(1);">Logout</a></li>
+                </ul>
+            </div>
+
+        </div>
+
+        <div class="container-header-footer">
+            <nav class="mobile-header">
+                <div class="icon1">
+                    <img src="images/mobil_menu.png" alt="hamburger menu" onClick="menuOpen(1)" />
+                </div>
+                <div class="logo">
+                    <img src="./images/logo_bogota.jpg" alt="logo" />
+                </div>
+                <ul class="menu-deployed">
+                    <li><a href="./about2.html " onClick="menuClose(1);">About</a></li>
+                    <li><a href="#program " onClick="menuClose(1);">Program</a></li>
+                    <li><a href="#speakers " onClick="menuClose(1);">Heros</a></li>
+                    <li><a href="#sponsors" onClick="menuClose(1);">Sponsor</a></li>
+                </ul>
+            </nav>
+        </div>
+
+    </header>
+
+    <div id="mobile_menu">
+        <div id="mobile_button" class="close_button" onClick="menuClose(1);">
+            <img src="images/mobil_close_menu.png" alt="close button" />
+        </div>
+        <ul>
+            <li><a href="./about2.html " onClick="menuClose(1);">About</a></li>
+            <li><a href="#program " onClick="menuClose(1);">Program</a></li>
+            <li><a href="#speakers " onClick="menuClose(1);">Heros</a></li>
+            <li><a href="#sponsors" onClick="menuClose(1);">Sponsor</a></li>
+        </ul>
+
+    </div>
+
+
+    <main>
+        <section id="cover" class="cover">
+            <div class="container">
+                <h2>"Hello! make it amazing!"</h2>
+                <h1 class="home">CREATING HIGH-PERFORMANCE
```

Comment:
- [ ] [OPTIONAL] Please consider making sure that the headings(h1-h6) follow a particular order. In this case, the `h1` should come before the `h2`